src/DashboardUI/src/components/map/Map.tsx

```diff
@@ -652,23 +652,62 @@ function Map({
             const mapContainer = document.getElementById('map');
 
             if (!mapContainer) {
-              throw new Error('Map container not found');
-              reject();
+              reject(new Error('Map container not found'));
+              return;
+            }
+
+            // resize component for standard size screenshot
+            mapContainer.style.width = options.width + 'px';
+            mapContainer.style.height = options.height + 'px';
+            const mapContainerContainsHeightClass = mapContainer.classList.contains('h-100');
+            if (mapContainerContainsHeightClass) {
+              mapContainer.classList.remove('h-100');
             }
```

Comment:
Meh, I don't think you need to check if it exists
```suggestion
            mapContainer.classList.remove('h-100');
```

Reply:
updating